core/tauri/permissions/.gitignore

```
@@ -1 +1 @@
-.schema.json
+*/schemas/schema.json
```

the schema files are in `permissions/window/.schema.json` and so on, shouldn't this be
```suggestion
.schema.json
```

---

i guess i forgot to delete some